templates/hub.html

```diff
@@ -35,7 +35,7 @@ <h1 class="u-mt-m">{{ content.title }}</h1>
         <p class="u-mb-no" data-qa="guidance">{{ content.guidance }}</p>
       </div>
     {% endif %}
-    {% if content.individual_response_url %}
+    {% if content.individual_response_url and not content.survey_complete %}
```

Comment:
What if `individual_response_url` wasn't set when survey was complete?

Reply:
As survey complete is only used for IR at the moment, and it adds logic in the template I think not setting  individual_response_url makes more sense. However not something I will block on

Reply:
I've updated it now as probably better to use less logic